src/SourceLocator/Type/AbstractSourceLocator.php

```diff
@@ -27,7 +28,7 @@
      * @param Identifier $identifier
      * @return LocatedSource
```

Comment:
Fixing broken documentation, `createLocatedSource` has always been able to return `null` so doc was wrong